anesthetic/plot.py

```diff
@@ -402,17 +402,18 @@ def scatter_plot_2d(ax, data_x, data_y, *args, **kwargs):
 
     Returns
     -------
-    lines: matplotlib.lines.Line2D
-        A list of line objects representing the plotted data (same as
-        matplotlib matplotlib.axes.Axes.plot command)
+    matplotlib.collections.PathCollection object
+        A PathCollection object representing the plotted data (same as
+        matplotlib matplotlib.axes.Axes.scatter command)
 
     """
     xmin = kwargs.pop('xmin', None)
     xmax = kwargs.pop('xmax', None)
     ymin = kwargs.pop('ymin', None)
     ymax = kwargs.pop('ymax', None)
+    s = kwargs.pop('s', 3)
```

Comment:
I assume that this default change is necessary because the default size is too large. Does ``markersize=1`` roughly correspond to ``s=3``?

Reply:
roughly, yes, that's why I picked it.